### ISSUE TITLE
[Dungeon][Occulus] Unable to cancel channeled drake abilities while on drake

### ISSUE THREAD

**Issue by reetarded** [Stormforge-gg/Bugtracker-Fordring#1123]
As title states, you are unable to cancel the channels of the green/bronze drake any way. Once you press the channel it will fully complete no matter if you try to move, cast a different ability, or repress the same channel button.

**Comment by slicplaya** [comment]
YES! It's almost impossible to heal with a green drake because it can kill you potentially. I hope this gets fixed before Ruby Sanctum is released lol..
